test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/CreateFineTuningJobRequest.cs

```diff
@@ -107,7 +107,7 @@ internal CreateFineTuningJobRequest(string trainingFile, string validationFile,
         /// </summary>
         public CreateFineTuningJobRequestModel Model { get; }
         /// <summary> The hyperparameters used for the fine-tuning job. </summary>
-        public CreateHyperparameters Hyperparameters { get; set; }
+        public CreateFineTuningJobRequestHyperparameters Hyperparameters { get; set; }
```

Comment:
This is an anonymous model: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/fine-tuning/models.tsp#L250